partner_identification_dea/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This Module is Manage the Medical License, DEA Number and its Expired Date for Contacts
```

Comment:
This sentence is not completely correct English. Make it something like:
This module ican be used to manage the Medical License and the DEA Number for contacts. This includes an expiry date and an automatic notification 30 days before the Medical License or DEA expires.